scss/_objects.grid.scss

```diff
@@ -4,47 +4,76 @@
     Grid system objects, works together with utility width classes
 */
 
+
+// ---
+// Variables
+// ---
+
+$grid-max-width : map-get($breakpoints,bp5) !default;
+$grid-gutter    : $columns-gutter !default;
+
+
+// ---
+// Styles
+// ---
+
+
+// Max width container
 .o-container {
     margin-left: auto;
     margin-right: auto;
     max-width: $grid-max-width;
 }
 
-// No gutters
 
 .o-grid {
-    @include clearfix;
+    display: flex;
+    align-items: stretch;
+    flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
@Dietr I think `flex-flow: row wrap;` is actually the default flow, so we can remove this, no?